aten/src/ATen/native/Blas.cpp

```diff
@@ -82,23 +90,17 @@ TORCH_IMPL_FUNC(addmv_out_cpu)(const Tensor &self, const Tensor &mat, const Tens
   }
 }
 
-Tensor &mv_out(const Tensor &self, const Tensor &vec, Tensor& result) {
+TORCH_IMPL_FUNC(mv_out)(const Tensor &self, const Tensor &vec,  const Tensor& result) {
   //self arg sent to addmv_out cannot be resized
   //here we use result as self argument for addmv, and result is user supplied and can be wrong size
   //it's not a hard error, because we allow resizing result, but it becomes a hard error
   //in addmv, because addmv expects self to satisfy proper conditions
   //to avoid this, supply correctly sized self, its contents doesn't matter because beta is 0
   if (result.dim() > 1 || (result.numel() != self.size(0) || result.numel() !=1)) {
     Tensor self_addmv = at::empty({self.size(0)}, self.options());
-    return at::addmv_out(result, self_addmv, self, vec, 0, 1);
+    at::addmv_out(const_cast<Tensor&>(result), self_addmv, self, vec, 0, 1);
```

Comment:
`at::cpu::addmv_out` here to avoid dispatch? It probably works for cuda inputs also